src/routes/jira/server/jira-app-creation-post.ts

```diff
@@ -0,0 +1,24 @@
+import e, { NextFunction, Request, Response } from "express";
+import { GitHubServerApp } from "models/github-server-app";
+
+export const JiraAppCreationPost = async (
+	req: Request,
+	res: Response,
+	next: NextFunction
+): Promise<e.Response<number, Record<string, string>> | void> => {
```

Comment:
Why is this route function returning?  All route functions should just return `Promise<void>`

Reply:
Huh weird. I wasn't getting an error here yesterday in my terminal and the only way to resolve it was to return the response but that doesn't appear to be happening now. :wat: